genson-java-datetime/README.md

```diff
@@ -0,0 +1,94 @@
+|----------------|----------|---------------------|---------------------|-----------------------------------------------------------------------|--------|
+| Instant        |  MILLIS  | Epoch milliseconds  |  Epoch Nano seconds | [second, nano]                                        | Second resolves to INSTANT_SECONDS which is number of epoch seconds |
+| ZonedDateTime  |  MILLIS  | Epoch milliseconds  |  Epoch Nano seconds | [year, month, day, hour, minute, second, nano, zoneId]        | |
+| OffsetDateTime |  MILLIS  | Epoch milliseconds  |  Epoch Nano seconds | [year, month, day, hour, minute, second, nano, offsetSeconds]  | |
+| LocaDateTime   |  MILLIS  | Epoch milliseconds  |  Epoch Nano seconds | [year, month, day, hour, minute, second, nano]                |  Milliseconds and nanos are obtained by converting LocalDateTime to ZonedDateTime using configured ZoneId |
+| LocalDate      |  MILLIS  | Epoch day           |  Epoch Day          | [year, month, day]                                                    | Millis and nanos both resolve to epoch day |
+| LocalTime      |  MILLIS  | Milliseconds of day | Nanoseconds of day  | [hour, minute, second, nano]                                                    |  |
+| Year           |  ARRAY   | Year                |  Year               | [year]                                                    |  |
+| YearMonth      |  ARRAY   | Epoch month         |  Epoch month        | [year, month]                                                    |  Epoch month is number of months from January 1970 |
+| MonthDay       |  ARRAY   | Not Supported       |  Not supported      | [month, day]                                               | Numeric timestamp not supported. Cannot use day of year due to possible leap year |
+| OffsetTime     |  ARRAY   | Not supported       |  Not supported      | [hour, minute, second, nano, offsetSeconds]                                                    |  Numeric timestamp not supported |
+| Period         |  ARRAY   | Not supported       |  Not supported      | [years, months, days]                                                    |  Numeric timestamp not supported |
+| Duration       |  ARRAY   | Not supported       |  Not supported      | [seconds, nanos]                                                    |  Numeric timestamp not supported |
+
+#### Annotations
+##### @JsonDateFormat
+The bundle is configured to work with the existing `@JsonDateFormat` annotation. The value of the annotation will be used as
+the pattern to construct a DateTimeFormatter. 
+
+The `asTimeInMillis` property is supported but does not function exactly the same.  The value only 
+determines whether the bundle will serialize/deserialize in the assigned TimestampFormat. It does not guarantee that the timestamp
+format is actually milliseconds.  Use `@TimestampFormat` to control the desired format
```

Comment:
Is the annotation `@TimestampFormat` or `@JsonTimestampFormat` as the following paragraph suggests?

Reply:
The timestamp is indeed `@JsonTimestampFormat `.  I have updated the documentation